composer.json

```diff
@@ -8,9 +8,18 @@
             "email": "szymon.kulczynski@sylius.com"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/Sylius/SyliusShopBundle.git"
+        }
+    ],
     "require": {
         "sylius/shop-bundle": "^1.7"
     },
+    "require-dev": {
+        "friendsoftwig/twigcs": "^v4.0-BETA4"
```

Comment:
Is there no stable version available?

Reply:
No. Soon i hope, but not now https://packagist.org/packages/friendsoftwig/twigcs and v3 is not very good.

Reply:
I'm not sure if it's such a great idea to add these dev dependencies to the `composer.json` file of the theme to be honest. Also not sure about adding unstable versions. It's nice that we can check the twig templates though!

Reply:
v4.0.0 is available :)

Reply:
Oh yeah !!! :heart: